### ISSUE TITLE
Build corretto on Xcode 9+ so that app built with corretto can be notarized

### ISSUE THREAD

**Issue by Sabir001** [corretto/corretto-8#153]
### Is your feature request related to a problem?
Yes. I want to publish Mac app with embedded JRE. But, to publish, I need to notarize app. Notarization needs executable files to be built with SDK 10.9 or plus. But, as corretto is built with Xcode 4, it is built with SDK 10.8.

### Describe a solution you would like
Built with xcode 9+. 

### Describe alternatives you have considered
My alternative is to build OpenJDK myself. But, that would be very tough job.

### Additional info
You can visit this repo: https://github.com/stooke/jdk8u-xcode10

**Comment by ziyiluo** [comment]
@Sabir001 Thanks for reaching out and you are right: This is a tough task :) Let me explain why it's hard and why we are prioritizing other tasks:
* Updating from Xcode 4 to Xcode 9+ is not only an update of the MacOS SDK version, but also a switch from the GCC tool chain to the Clang tool chain. The latter is a bigger headache. In my experience, a lot of common (or say, platform-independent) code, makefiles, configurations need to be updated, which means the change cannot be isolated to Mac-specific code.
* For us, updating to Xcode 9+ would break the system compatibility we committed. Currently, [Corretto 8 can be installed on MacOS version >= 10.10](https://docs.aws.amazon.com/corretto/latest/corretto-8-ug/macos-install.html). Updating Xcode version will bump the minimum system requirement to 10.13.

In summary, it is in our backlog but we'd like to gather more data points on how Corretto 8 Mac users can benefit from this update.

**Comment by ziyiluo** [comment]
Btw, have you considered migrating your app to adopt [Corretto 11](https://github.com/corretto/corretto-11)? Corretto 11 can be built with the latest MacOS tool chain.

**Comment by Sabir001** [comment]
Well, initially I planned to use [corretto 11](https://github.com/corretto/corretto-11) actually. But, as most of the libraries are not yet modularised, it is tough to create an executable with embedded JRE where there are lots of dependencies on other libraries.

**Comment by alvdavi** [comment]
Hi @Sabir001, we understand your problem, and we hope in the future you are able to work with Corretto11 once those libraries are properly updated. However, as my colleague answered, it is not feasible to change the xcode version we use.

**Comment by JonasWilms** [comment]
What happens if you use XCode 10+ with the macosx-version-min flag to support 10.10?
It might be slower on newer macOS version, but at least it would run on macOS 10.15+ after January 2020.

Even Oracle changes it's build system to XCode 10+
https://bugs.openjdk.java.net/browse/JDK-8226288
